server/plugin/api.go

```diff
@@ -458,13 +491,13 @@ func (p *Plugin) handleDriveWatchNotifications(c *Context, w http.ResponseWriter
 	}
 	changeList, err := srv.Changes.List(pageToken).Fields("*").Do()
 	if err != nil {
-		p.API.LogError("Failed to fetch google drive changes", "err", err)
+		p.API.LogError("Failed to fetch Google Drive changes", "err", err)
```

Comment:
I like the capitalization for proper names but would prefer it be consistent across all the log lines.